gradle/libs.versions.toml

```diff
@@ -0,0 +1,151 @@
+[versions]
+accompanist = "0.30.1"
+activity-compose = "1.7.2"
+androidBeaconLibrary = "2.19.6"
+androidJunit5 = "1.9.3.0"
+androidPlugin = "8.0.2"
+appcompat = "1.6.1"
+biometric = "1.1.0"
+blurView = "version-1.6.6"
+car-versions = "1.3.0-rc01"
+changeLog = "3.4"
+community-material-typeface = "7.0.96.0-kotlin"
+compose-bom = "2023.06.01"
+compose-compiler = "1.4.8"
+composeFoundation = "1.1.2"
+converterJackson = "2.9.0"
+coreKtx = "1.10.1"
+cronet-embedded = "113.5672.61"
+emojiJava = "5.1.1"
+exoplayer = "2.19.0"
+firebase-bom = "32.2.0"
+firebaseAppdistributionGradle = "4.0.0"
+fragment-ktx = "1.6.1"
+constraintlayout = "2.1.4"
+googleServices = "4.3.15"
+guava = "32.1.1-android"
+healthServicesClient = "1.0.0-rc01"
+hilt = "2.47"
+iconics = "5.4.0"
+jackson-module-kotlin = "2.13.5"
+kotlinx-coroutines = "1.7.2"
+kotlin = "1.8.22"
+ktlint = "11.5.0"
+lifecycleViewmodelKtx = "2.6.1"
+loggingInterceptor = "4.11.0"
+navigation-compose = "2.6.0"
+okhttp = "4.11.0"
+material = "1.9.0"
+play-services-threadnetwork = "16.0.0"
+play-services-home = "16.0.0"
+play-services-location = "21.0.1"
+picasso = "2.8"
+play-services-wearable = "18.0.0"
+playPublisher = "3.8.4"
+preference-ktx = "1.2.0"
+recyclerview = "1.3.1"
+lifecycle-runtime-ktx = "2.6.1"
+reorderable = "0.9.6"
+retrofit = "2.9.0"
+roomKtx = "2.5.2"
+roomRuntime = "2.5.2"
+watchfaceComplicationsDataSourceKtx = "1.1.1"
+wear = "1.2.0"
+wear-tiles = "1.1.0"
+wearPhoneInteractions = "1.0.1"
+wearInput = "1.2.0-alpha02"
+webkit = "1.7.0"
+wear-remote-interactions = "1.0.0"
+sentry-android = "6.26.0"
+workRuntimeKtx = "2.8.1"
+
+[plugins]
+ktlint = { id = "org.jlleitschuh.gradle.ktlint", version.ref = "ktlint" }
+
+[libraries]
+accompanist-themeadapter-material = { module = "com.google.accompanist:accompanist-themeadapter-material", version.ref = "accompanist" }
+accompanist-systemuicontroller = { module = "com.google.accompanist:accompanist-systemuicontroller", version.ref = "accompanist" }
+activity-compose = { module = "androidx.activity:activity-compose", version.ref = "activity-compose" }
+activity-ktx = { module = "androidx.activity:activity-ktx", version.ref = "activity-compose" }
+android-beacon-library = { module = "org.altbeacon:android-beacon-library", version.ref = "androidBeaconLibrary" }
+android-plugin = { module = "com.android.tools.build:gradle", version.ref = "androidPlugin" }
+android-junit5 = { module = "de.mannodermaus.gradle.plugins:android-junit5", version.ref = "androidJunit5" }
+androidx-compose-foundation = { module = "androidx.wear.compose:compose-foundation", version.ref = "composeFoundation" }
+androidx-compose-material = { module = "androidx.wear.compose:compose-material", version.ref = "composeFoundation" }
+androidx-compose-navigation = { module = "androidx.wear.compose:compose-navigation", version.ref = "composeFoundation" }
```

Comment:
These names aren't helpful, what would the different be between compose-foundation and androidx-compose-foundation? Suggestion: make it something like wear-compose-foundation.